pageStyles/candidates.module.scss

```diff
@@ -0,0 +1,43 @@
+﻿@import 'pageStyles/constants';
+
+.link {
+  margin: unset;
+}
+
+.desktopButton {
+  background-color: $softwireYellow;
+  border: none;
+  padding: 5px 15px;
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 10px;
+  position: fixed;
+  display: none;
+  
+  :hover{
+    text-decoration: underline;
+    cursor: pointer;
+  }
+    
+  @media (min-width: 768px) {
+    right: 10px;
+    top: 265px;
```

Comment:
this seems large and fragile! 😂

I'm guessing this is measured from the top of the page right now?
Looks from the design like we want it to be positioned next to the 'Select a candidate' title / in that title section.

So I'd try writing the CSS to position it based on where that is - eg absolutely position it based on that as a parent?
This should
- make it less fragile: at the moment if we change anything then we'd need to reposition this button
- make it easier to understand - much clearer where you are trying to place it.

Reply:
Hopefully less fragile now, positioned absolute relative to parent